docs/contributing.rst

```diff
@@ -13,6 +13,9 @@ want to review in order to get started.
 
 Also, feel free to join the discussions on the `developer mailing list`_.
 
+If you want a quick guide to getting your development enviroment setup, please
+use `the contributing instructions on GitHub`_.
```

Comment:
I got 404 page when going by the link. Please, check if it really works.

Reply:
It should work after this PR, which moves onboarding/onboarding.md to contributing/contributing.md

Reply:
yep, it works